backend/open_webui/retrieval/vector/dbs/qdrant_multitenancy.py

```
@@ -0,0 +1,365 @@
+import logging
+from typing import Optional, Tuple, List, Dict, Any
+from urllib.parse import urlparse
+
+import grpc
```

Ruff (0.12.2)

`grpc` imported but unused

Remove unused import: `grpc`

(F401)